NebulaHost/PacketProcessors/Routers/PlanetBroadcastProcessor.cs

```diff
@@ -0,0 +1,32 @@
+﻿using NebulaModel.Attributes;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Processors;
+using NebulaModel.Packets.Routers;
+
+namespace NebulaHost.PacketProcessors.Routers
+{
+    [RegisterPacketProcessor]
+    class PlanetBroadcastProcessor : IPacketProcessor<PlanetBroadcastPacket>
+    {
+        private PlayerManager playerManager;
+        public PlanetBroadcastProcessor()
+        {
+            playerManager = MultiplayerHostSession.Instance.PlayerManager;
+        }
+        public void ProcessPacket(PlanetBroadcastPacket packet, NebulaConnection conn)
+        {
+            Player player = playerManager.GetPlayer(conn);
+            if (player != null)
+            {
+                //Forward packet to other users
+                playerManager.SendRawPacketToPlanet(packet.PacketObject, packet.PlanetId, conn);
```

Comment:
Wouldn't it be better to include the host in this call rather than individually below?

Reply:
you are right, I should do the check for the same planet in the specific packet processor and not here. I will fix